scanapi/reporter.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+from jinja2 import Environment, PackageLoader, select_autoescape
+import logging
+
+from scanapi.settings import SETTINGS
+
+logger = logging.getLogger(__name__)
+
+
+class Reporter:
+    def __init__(self, docs_path, reporter, template):
+        self.docs_path = docs_path
+        self.reporter = reporter
+        self.template = template
+
+    def write(self, responses):
+        logger.info("Writing documentation")
+
+        env = Environment(loader=PackageLoader("scanapi", "templates"))
+        template = env.get_template(f"{self.reporter}.jinja")
+        content = template.render(responses=responses)
+
+        if self.reporter == "console":
+            print(f"\n{content}")
+            return
+
+        if self.docs_path is None:
+            outputs = {"html": "docs.html", "markdown": "docs.md"}
```

Comment:
And here it could be `report.html` and `report.md`

Reply:
Let's google a little bit and see what other tools use as their default naming convention for report files. My only concern is changing a file that was already there being used for something else.

Reply:
Super good point. We can rename it to `scanapi-report` to ensure that no conflicts will happen. What do you think?

Reply:
Done. Now we have `scanapi-report.html` and `scanapi-report.md` as main files.

Reply:
🚀